tests/store/test_file_store.py

```diff
@@ -285,6 +286,33 @@ def test_list_run_infos(self):
                 dict_run_info['lifecycle_stage'] = LifecycleStage.ACTIVE
                 self.assertEqual(dict_run_info, dict(run_info))
 
+    def test_log_metric_allows_multiple_values_at_same_ts_and_run_data_uses_max_ts_and_value(self):
+        fs = FileStore(self.test_root)
+        run_uuid = self._create_run(fs).info.run_uuid
+
+        metric_name = "test-metric-1"
+        timestamp_low = 1000
+        timestamp_high = 2000
+        value_range = map(float, range(-10, 10))
```

Comment:
to sid's point, might be more re-assuring if `timestamp_low` had one value greater than those of `timestamp_high`

Reply:
Good point! I changed the range for `timestamp_low` to `(-20,20)` and kept the range for `timestamp_high` at `(-10,10)`.